sparsity/dask/core.py

```diff
@@ -59,6 +59,36 @@ def finalize(results):
     return sp.SparseFrame.vstack(results)
 
 
+def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
+    for df in dfs:
+        if not isinstance(df, Delayed):
+            raise TypeError("Expected Delayed object, got %s" % type(df).__name__)
+
+    if meta is None:
+        meta = delayed(make_meta)(dfs[0]).compute()
+    else:
+        meta = make_meta(meta)
+
+    name = prefix + "-" + tokenize(*dfs)
+    dsk = merge(df.dask for df in dfs)
```

Comment:
Sure you can use `dask` attribute here? I'm not sure if that is not deprecated by dask collection and the latest API suggests to use `.__dask_graph__()`

Reply:
Dask is using it internally all the time. This is just copied.

Reply:
I also didn't find information that it's deprecated.